Description: Disable ieee_exceptions.F95 due to build failures under gfortran
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2018-11-29
Forwarded: no

Index: flang-20180921/runtime/flang/CMakeLists.txt
===================================================================
--- flang-20180921.orig/runtime/flang/CMakeLists.txt
+++ flang-20180921/runtime/flang/CMakeLists.txt
@@ -452,9 +452,9 @@ FOREACH(file ${FTN_SUPPORT_DESC_DEP})
 ENDFOREACH(file)
 
 LIST(APPEND FTN_SUPPORT_DESC_DEP ieee_arithmetic.F95)
-LIST(APPEND FTN_SUPPORT_DESC_DEP ieee_exceptions.F95)
+#LIST(APPEND FTN_SUPPORT_DESC_DEP ieee_exceptions.F95)
 LIST(APPEND FTN_SUPPORT_I8 "${I8_FILES_DIR}/ieee_arithmetic.F95")
-LIST(APPEND FTN_SUPPORT_I8 "${I8_FILES_DIR}/ieee_exceptions.F95")
+#LIST(APPEND FTN_SUPPORT_I8 "${I8_FILES_DIR}/ieee_exceptions.F95")
 
 add_flang_library(flang_static
   ${FTN_INTRINSICS_DESC_INDEP}
@@ -541,7 +541,7 @@ set_source_files_properties(
 # State a dependency on the module
 set_source_files_properties(
   ieee_arithmetic.F95
-  ieee_exceptions.F95
+#  ieee_exceptions.F95
   PROPERTIES
   OBJECT_DEPENDS ${CMAKE_Fortran_MODULE_DIRECTORY}/iso_c_binding.mod
   )
@@ -549,7 +549,7 @@ set_source_files_properties(
 # State a dependency on the module
 set_source_files_properties(
   ${I8_FILES_DIR}/ieee_arithmetic.F95
-  ${I8_FILES_DIR}/ieee_exceptions.F95
+#  ${I8_FILES_DIR}/ieee_exceptions.F95
   PROPERTIES 
   COMPILE_DEFINITIONS DESC_I8
   OBJECT_DEPENDS ${CMAKE_Fortran_MODULE_DIRECTORY}/iso_c_binding.mod
